### PR TITLE
Add rebaler

### PR DESCRIPTION
**Pull request by druvus**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/rebaler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+build:
+  number: 0
+  skip: True  # [py27]
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - biopython
```
Comment (mbargull) [inline recipes/rebaler/meta.yaml]: `biopython` can be removed from the `build` requirements.

File: recipes/rebaler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/rrwick/Rebaler/archive/v{{ version }}.tar.gz
+  fn: rebaler_{{ version }}.tar.gz
+  sha256: c91bb08786fd5ba851c92111b6e0bb95f93ba8d6feeef57c283bb2d67da4cc93
+
+build:
+  number: 0
+  skip: True  # [py27]
```
Comment (mbargull) [inline recipes/rebaler/meta.yaml]: Can you add
```yaml
  entry_points:
    - rebaler = rebaler.rebaler:main
```
to the `build:` section?

File: recipes/rebaler/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - biopython
+
+    run:
+        - python
+        - setuptools
```
Comment (mbargull) [inline recipes/rebaler/meta.yaml]: `setuptools` can be removed from `run` requirements.

**Comment by druvus** [comment]
thanks @mbargull  & @cbrueffer

**Comment by mbargull** [comment]
And to you!

**Pull request merged**